thorlcr/graph/thgraph.cpp

```diff
@@ -1489,6 +1489,32 @@ void CGraphBase::end()
     }
 }
 
+void CGraphBase::addThorDebugProbe(const char *edgeId, IThorDataLink *p)
+{
+    if (!thorDataLinkTracingProbe.get())
+        thorDataLinkTracingProbe.setown(new MapStringToIThorDataLink);
+
+    thorDataLinkTracingProbe->setValue(edgeId,p);
+}
+
+IThorDataLink *CGraphBase::queryThorDataProbe(const char *edgeId)
+{
+    if (thorDataLinkTracingProbe.get())
+    {
+        IThorDataLink **p = thorDataLinkTracingProbe->getValue(edgeId);
+        if (p && *p) return *p;
+    }
+
+    return NULL;
+};
+
+void CGraphBase::removeThorDataProbe(const char *edgeId)
+{
+    if (thorDataLinkTracingProbe.get())
+        thorDataLinkTracingProbe->remove(edgeId);
+};
+
+
```

Comment:
Are addThorDebugProbe()/queryThorDataProbe() and removeThorDataProbe() redundant now?
I don't see them used anywhere.